### ISSUE TITLE
Logic in mesh_sequence._output()

### ISSUE THREAD

**Issue by stephankramer** [pyroteus/goalie#93]
https://github.com/pyroteus/goalie/blob/8a9cd9b26027b2a9145146e27371dc584d34bec7/goalie/mesh_seq.py#L437C29-L437C29

Just a quick query: what is the rationale to allow _output() to return None if it can't find the output on the given block with the name it expects? Why is this allowed at all, and only in the steady case? As the comment says it relies on the user using the correct name in get_solver - but if they get it wrong there is no error (in the steady case).

**Comment by jwallwork23** [comment]
Yeah you're right, this is probably wrong and at best misleading.
